ogr/ogrsf_frmts/filegdb/FGdbLayer.cpp

```diff
@@ -4014,6 +4014,9 @@ int FGdbLayer::TestCapability( const char* pszCap )
     else if( EQUAL(pszCap,OLCIgnoreFields) )
         return TRUE;
 
+    else if( EQUAL(pszCap,OLCCurveGeometries) )
```

Comment:
I don't think this is appropriate for the FileGDB driver. It uses internally OGRWriteToShapeBin() which isn't ready yet to support curve geometries (it could, just that it isn't implemented).

Reply:
done